### DISCUSSION TITLE
Next up for NeoEloquent?

### DISCUSSION THREAD

**Discussion by berteltorp** [ulobby/NeoEloquent#18] [General]
Talking points for status meeting

1. Driver - which driver should we using going forward?
2. Objective: Support the most recent version of Laravel
3. Objective: Support the most recent version of Neo4j (esp. cluster-support)
4. 🧹 House keeping - stale branches
5. PR's to upstream
6. Releases not registered in packagist

**Comment by berteltorp** [comment]
@harris21 @SimonThordal @joseraul @Johnhundred  - Let's do a status call on NeoEloquent (attendance optional) to decide on the above 😄

**Reply by joseraul** [reply]
yessss, in fact, I wanted to point out the releases stuff this morning in the stand-up, but the server issues had definitely a higher priority.

**Reply by berteltorp** [reply]
Figured out the release issue the naming was off @joseraul 😄 

Was: "v.6.0.2
Should have been "v6.0.2"

**Reply by joseraul** [reply]
Let's talk about this tomorrow, I'm banging my head against the wall 😅

**Comment by berteltorp** [comment]
New versioning scheme: 
Laravel major = NeoEloquent major + rest semver

**Comment by harris21** [comment]
Laravel 7 support added. We'll soon add support for Laravel 8 as well.

**Comment by harris21** [comment]
Laravel 8 support added.

**Comment by harris21** [comment]
New Neo4J client, with cluster support, worth looking into :)

https://medium.com/neo4j/connect-to-neo4j-with-php-e10e24afedff

@berteltorp @SimonThordal @joseraul @Johnhundred
